src/constants/Routes.js

```diff
@@ -0,0 +1,8 @@
+export const SETTINGS = 'SETTINGS';
```

Comment:
Please rename all the routes in pattern like *routeName*_SCREEN
F.e. SETTINGS_SCREEN